.github/workflows/release.yml

```diff
@@ -83,11 +83,11 @@ jobs:
     steps:
       - name: Install Protoc
         uses: arduino/setup-protoc@v1
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
           node-version: '10'
-          registry-url: 'https://registry.npmjs.org'
-      - uses: actions/checkout@v1
+          registry-url: 'https://registry.npmjs.org/'
```

Comment:
`registry-url` is must needed?

Reply:
It is optional field, I am sorry that I couldn't find any document indicating the default value of this field.

But seems the official document set this value without `/` in the end, so the publish issue may cause by another mistake.
https://github.com/actions/setup-node/blob/d08cf222111d5c1d21b3cd4b958937f818d10d9a/docs/advanced-usage.md

Reply:
I see.
btw, can you publish packages which have been failed to publish?

Reply:
And, please sign CLA :)

Reply:
How is going?